vectorizer.py

```diff
@@ -83,9 +84,17 @@ class SentenceTransformerVectorizer:
     cuda_core: str
 
     def __init__(self, model_path: str, model_name: str, cuda_core: str):
+        local_files_only = os.getenv("LOCAL_FILES_ONLY", "False").lower() in (
```

Comment:
can you move this logic that gets this setting to `config.py`? you can follow how we introduced `TRUST_REMOTE_CODE` setting